### DISCUSSION TITLE
En reports no se puede descargar el pdf de una respuesta.

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#133] [4. Bug Reports] [answered]
Hola

Quería probar descargar el pdf de la respuesta de un formulario. SIn embargo en el momento de hacerlo, me carga una ventana nueva y no pasa nada. Adjunto video para que lo puedan ver:

**Link video:** https://drive.google.com/file/d/1nL7amW4MX-6LVNq-wCx4zKFmzEqDXhhp/view?usp=sharing

Quedo atento
Gracias

**Comment by acedward** [comment, marked as answer]
Confirmado, es un bug.
Gracias por reportar.

**Reply by NicolasPLG** [reply]
:)
